descriptors/ModuleDescriptor-template.json

```diff
@@ -540,6 +540,18 @@
             "finance-storage.transactions.item.put"
           ]
         },
+        {
```

Comment:
Please increase version of the API to "4.2"

Reply:
Done. Thanks.